testsuite/gnat2goto/tests/primitive_pointer/test.opt

```diff
@@ -0,0 +1 @@
+ALL XFAIL gnat2goto fails to compile input (regression)
```

Comment:
GitHub is showing newlines are missing from the end of a few files.

Reply:
Do these files actually need a newline at the end? They are `.opt`.

Reply:
Newlines at the end of files are just generally 'good style', not just for particular file types.